xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/pvr/ChannelGroups.h

```diff
@@ -36,12 +36,12 @@ namespace addon
 ///@{
 class PVRChannelGroup : public CStructHdl<PVRChannelGroup, PVR_CHANNEL_GROUP>
 {
+friend class CInstancePVRClient;
```

Comment:
From code guidlines two spaces before, here and other places.

Reply:
Sure, will fix that. Thanks.